pontoon/homepage/static/css/homepage.css

```diff
@@ -2,19 +2,6 @@ html {
   font-size: 8.22px;
 }
 
-@media only screen and (min-width: 768px) {
-  html {
-    font-size: 0.6320644vw;
-  }
-}
-
-@media only screen and (min-width: 1500px) {
-  html {
-    font-size: 10.44px;
-  }
-}
-
-
```

Comment:
As the different screen resolutions was facing issues with the font-size, I used them as they were working as the ratio system for the other screens. Now, it will be the same for all the screen sizes.